### PR TITLE
Unity and caliper test

### PR DESCRIPTION
**Pull request by acdpsn**
Has two buttons.
The first attemps to send a caliper event to https://lti.tools/caliper/event?key=milk
The second attempts to send a POST request to https://3eec5ec8-3d63-4c36-9789-52475ebc46d2.mock.pstmn.io/simpledata

### CODE REVIEW

File: Unity/2019.1/Events with Caliper/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+2019-05-29 09:22:10.976383-0400 EventsWithCaliper[2157:1463611] [BoringSSL] nw_protocol_boringssl_get_output_frames(1301) [C2.1:2][0x10db0d3b0] get output frames failed, state 8196
+2019-05-29 09:22:10.976469-0400 EventsWithCaliper[2157:1463611] TIC Read Status [2:0x0]: 1:57
+2019-05-29 09:22:10.976478-0400 EventsWithCaliper[2157:1463611] TIC Read Status [2:0x0]: 1:57
+```
+
+I googled the -998 error and found a couple of threads discussing the issue. [1](https://forum.unity.com/threads/no-error-reported-on-ios.670207/) [2](https://www.reddit.com/r/Unity2D/comments/bkvtbz/post_request_with_binary_field_fails_on_ios/) The latter leads me to believe that the caliper library is using a method for creating async POST requests that is incompatible with iOS for some reason.
+
+I decided to look into alternate ways to use async and tried the code from [this article](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong#The-Fix-is-In). It worked. So a possible solution is to fix or rewrite the library to use the Client object.
```
Comment (pushyamig) [inline Unity/2019.1/Events with Caliper/README.md]: SO is this means you were able to send caliper event from IOS device to an Endpoint?

Comment (acdpsn) [reply Unity/2019.1/Events with Caliper/README.md]: No, this is still in the same state that it was in before, I had just forgotten to create a pull request. This readme is outdated so I'll update it.

File: Unity/2019.1/Events with Caliper/Assets/Plugins/caliper-net/Release/ImsGlobal.Caliper.dll.config [hunk 2]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-7.0.0.0" newVersion="7.0.0.0" />
```
Comment (pushyamig) [inline Unity/2019.1/Events with Caliper/Assets/Plugins/caliper-net/Release/ImsGlobal.Caliper.dll.config:7]: What is the purpose of this configuration?

File: Unity/2019.1/Events with Caliper/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿I am following the instructions from the [caliper-net example](https://github.com/IMSGlobal/caliper-net#getting-started).
```
Comment (pushyamig) [inline Unity/2019.1/Events with Caliper/README.md]: I see a lot of file changes here which some are made by unity as part of creating components in Unity. I think listing what files type are default created by Unity, what type is generated by when creating object/component for test cases in unity and files created for coding (like `.cs') files will be useful.

File: Unity/2019.1/Events with Caliper/Assets/Scripts/CaliperEventManager.cs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    {
+
+    }
+
+    void ActionButtonEvent()
+    {
+        TestCaliperEventAsync();
+    }
+
+    async void TestCaliperEventAsync() // should not return void
```
Comment (pushyamig) [inline Unity/2019.1/Events with Caliper/Assets/Scripts/CaliperEventManager.cs]: If we are not able to send the caliper event using the library then this code needs appropriate comments works from Unity Simulator and not from the Device

File: Unity/2019.1/Events with Caliper/README.md (hunk 3 above)
Comment (acdpsn) [reply Unity/2019.1/Events with Caliper/README.md]: I added a list of files i created to the readme

File: Unity/2019.1/Events with Caliper/Assets/Plugins/caliper-net/Release/ImsGlobal.Caliper.dll.config (hunk 2 above)
Comment (acdpsn) [reply Unity/2019.1/Events with Caliper/Assets/Plugins/caliper-net/Release/ImsGlobal.Caliper.dll.config:7]: This was generated by Visual Studio on Windows when I built the Caliper-net dll.

File: Unity/2019.1/Events with Caliper/Assets/Scripts/CaliperEventManager.cs (hunk 4 above)
Comment (acdpsn) [reply Unity/2019.1/Events with Caliper/Assets/Scripts/CaliperEventManager.cs]: I have added comments.

File: Unity/2019.1/Events with Caliper/README.md (hunk 1 above)
Comment (acdpsn) [reply Unity/2019.1/Events with Caliper/README.md]: The readme has been updated to reflect the current state of this project.

**Pull request merged**